src/common.jl

```diff
@@ -187,17 +215,14 @@ function _initialize_bus_data!(
         temp_bus_types[bus_no] = bt
     end
 
-    bus_type_priorities = Dict{PSY.ACBusTypes, Int}(
-        PSY.ACBusTypes.REF => 3,
-        PSY.ACBusTypes.PV => 2,
-        PSY.ACBusTypes.PQ => 1,
-    )
+    # FIXME handle combining buses more intelligently.
+    # perhaps move some of this logic into PNM
     for (bus_no, reduced_bus_nos) in bus_reduction_map
         # pick the "highest" bus type among the reduced buses, where REF > PV > PQ.
         corrected_bus_types =
             [temp_bus_types[reduced_bus_no] for reduced_bus_no in reduced_bus_nos]
         push!(corrected_bus_types, temp_bus_types[bus_no])
-        combined_bus_type = findmax(bt -> bus_type_priorities[bt], corrected_bus_types)[1]
+        combined_bus_type = findmax(bt -> BUS_TYPE_PRIORITIES[bt], corrected_bus_types)[1]
```

Comment:
The bus type values (REF, PV, PQ) are already numbers 1, 2, 3. So just the minimum of those three would be more simple. With the downside that we rely on the values of REF, PV, PQ staying the same.